README.md

```diff
@@ -49,6 +49,17 @@ a later release.
 
 Moreover if you adopt OCamlFormat in one project it will not break your workflow in your other projects. Indeed OCamlFormat modifies a file only if it can find an `.ocamlformat` file, so adding a save hook in your editor will only simplify your workflow in projects using OCamlFormat.
 
+### Using OCamlFormat without breaking `git blame`
+
+* Create `.git-blame-ignore-revs` in your project
+* Add revision(s) where the code was re-formatted
+```
+# Apply new formatting with OCamlFormat
+2ceaf76b9f84cb632327c1479d0f30acfa3eeba2
+```
+* Run `git config --local blame.ignoreRevsFile .git-blame-ignore-revs`
+* All future use of `git blame` will now provide the relevant change message
```

Comment:
For this line in particular, I wonder if it is as explicitly informative as possible. It seems that maybe something more explicit could be used instead of "provide" and/or "relevant". What happens is that git blame will omit the reformatting commits, and automatically blame further back into the history, correct? Is there a standard oneish-line explanation of such behavior?

Reply:
I slightly rephrased: `provide blame information omitting the reformatting commits`, if that's alright for you we can merge, that's a great addition to the documentation!

Reply:
The man for `git-blame` says:

> Ignore changes made by the revision when assigning blame, as if the change never happened. Lines that were changed or added by an ignored commit will be blamed on the previous commit that changed that line or nearby lines.

We could repeat the second sentence ?